cli/tauri.js/api-src/path.ts

```diff
@@ -8,7 +8,8 @@ import { BaseDirectory } from './fs'
  */
 async function appDir(): Promise<string> {
   return await promisified<string>({
-    cmd: 'getDirectory',
+    cmd: 'resolvePath',
+    path: '.',
```

Comment:
@lucasfernog this will add a "dot" to the end of the path, I believe an empty string is better.

Reply:
One more thing, if a base directory doesn't exist, the user will get an error like this `Path Error:unable to determine base dir path`. In my opinion this will confuse the user as he doesn't know the underlying api is just resolving a path using a base dir, instead it would be better to just return null like tauri's Path api in rust.